### ISSUE TITLE
Single Code Element Binding

### ISSUE THREAD

**Issue by robnist** [haffonist/igamt#1845]
Wondering if we can remove the [] when everything is all 1's. I think we do this in validation and this will be the case in > 98% of the time.

![image](https://user-images.githubusercontent.com/14874904/27138277-2af8ba40-50ee-11e7-8067-2ca3dc7af0cb.png)

**Comment by Jungyubw** [comment]
Yes I will hide Instance number in case of “1”. I have the ticket for this already.

Thanks..


Jungyub Woo



On Jun 14, 2017, at 10:44 AM, Rob <notifications@github.com<mailto:notifications@github.com>> wrote:

Wondering if we can remove the [] when everything is all 1's. I think we do this in validation and this will be the case in > 98% of the time.
